Examples/NoModules/Chapter 10/Ex10_02.cpp

```diff
@@ -0,0 +1,46 @@
+// Overloading function templates
+#include <iostream>
+#include <format>
+#include <string>
+
+template<typename T> T larger(T a, T b);    // Function template prototype
+template <typename T> T* larger(T*, T*);
+template <typename T> const T* larger(const std::vector<T>& data);
```

Comment:
My text says to add:
`template <typename T> T larger(std::span<const T> data);`
`template <typename T> T* larger(T*, T*);`
that is, span, while in your code it is vector.
did you update the text?

Reply:
Basically, I don't have the latest text.
The code I see here does not match my text.
But, I had a lot of comments in the text, so you might have changed the code.
PLease just double check that the code here matches the text.
(in the text for example, there was a string example, which I don't see here anymore.)

Reply:
but besides that, the code here compiles and runs.

Reply:
Text was updated after your review remarks, yes. Got rid of the spans, because that was just causing too many issues.